### ISSUE TITLE
Freezes when loading and resizing the training images...

### ISSUE THREAD

**Issue by ClaudeCoulombe** [wagonhelm/TF_ObjectDetection_API#2]
I've experienced a problem when loading and resizing the training images. It runs pretty fast and stucked on the image 46 of the training dataset. No visible error messages... but I remark that the notebook was disconnected. Probably a timeout...

I suspect memory problem. How much memory should be allocated to Docker?  

Thanks

**Comment by wagonhelm** [comment]
No issues with the links on my end, maybe check bash history as to why notebook disconnects?  You should have at the absolute least 4gb of ram allocated to docker.  Sorry I don't use docker much, maybe try native python and pip?

**Comment by Juary88** [comment]
hello，when i run this command 
%%bash 
rm -rf object_detection_graph
python3 models/research/object_detection/export_inference_graph.py \
    --input_type image_tensor \
    --pipeline_config_path data/ssd_mobilenet_v1_pets.config \
    --trained_checkpoint_prefix data/model.ckpt-4094 \
    --output_directory object_detection_graph

error:Traceback (most recent call last):
  File "models/research/object_detection/export_inference_graph.py", line 117, in <module>
    tf.app.run()
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/platform/app.py", line 48, in run
    _sys.exit(main(_sys.argv[:1] + flags_passthrough))
  File "models/research/object_detection/export_inference_graph.py", line 113, in main
    FLAGS.output_directory, input_shape)
  File "/notebooks/object/models/research/object_detection/exporter.py", line 427, in export_inference_graph
    input_shape, optimize_graph, output_collection_name)
  File "/notebooks/object/models/research/object_detection/exporter.py", line 353, in _export_inference_graph
    postprocessed_tensors = detection_model.postprocess(output_tensors)
  File "/notebooks/object/models/research/object_detection/meta_architectures/ssd_meta_arch.py", line 405, in postprocess
    class_predictions_without_background)
  File "/notebooks/object/models/research/object_detection/builders/post_processing_builder.py", line 94, in score_converter_fn
    scaled_logits = tf.divide(logits, logit_scale, name='scale_logits')
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/ops/math_ops.py", line 307, in divide
    return DivideDelegateWithName(x, name) / y
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/ops/math_ops.py", line 292, in __truediv__
    return _truediv_python3(self.x, y, self.name)
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/ops/math_ops.py", line 952, in _truediv_python3
    (x_dtype, y_dtype))
TypeError: x and y must have the same dtype, got tf.float32 != tf.int32

please tell me how to solve it?

**Comment by ClaudeCoulombe** [comment]
Thank you. I've tried with 8 GB, without success... Sadly, since I had got the same problem with native Python and pip that's why I wanted to try with docker hoping that was a configuration or dependence issue. I know that with time (which tends to be scarce) we would have succeeded. Anyway, with your nice tutorial I'm now more familiar with the TensorFlow ObjectDetection API. That was the main goal. Thank you for your support!
